documentation/milestones/individual_milestones/get_opportunities.md

```diff
@@ -79,17 +86,24 @@ The way that data is stored and delivered through the API should balance:
   - [ ] All code quality checks set up in the [Developer Tools milestone](https://github.com/HHS/grants-equity/issues/50) are passing
   - [ ] The resources required to deploy and host the API are provisioned programmatically using the [Infrastructure-as-Code milestone](https://github.com/HHS/grants-equity/issues/123) framework
   - [ ] Code changes are deployed using the CI/CD pipeline set up in [the Back-end CI/CD milestone](https://github.com/HHS/grants-equity/issues/57)
-  - [ ] Logging/monitoring is configured, and it both records the metrics defined below and alerts the development team when the API is down
+  - [ ] DB migrations are automatically configured through scripts that enable upgrading/downgrading database quickly and easily (e.g., by using Alembic https://github.com/sqlalchemy/alembic)
+  - [ ] The API has been load tested using the framework established in the [Peformance Testing Framework milestone](https://github.com/HHS/grants-equity/issues/69) to ensure that it remains performant under heavy user traffic
+  - [ ] Logging/monitoring is configured, and it both records the metrics defined below and alerts the development team when the API is down per the [API Logging & Monitoring milestone](https://github.com/HHS/grants-equity/issues/370)
+  - [ ] An incident response protocol is in place and the on-call team have followed that protocol in at least one training or simulation per the [Incident Response milestone](https://github.com/HHS/grants-equity/issues/373)
```

Comment:
Nit: we should probably call out creating alerts explicitly as well. (e.g., like alerts on a certain frequency of response code, or on the presence of some types of logs.) The logs enable the alerts, and the alerts are what will trigger the escalation process (i.e., PagerDuty) for incident response.

Reply:
I can adjust the language of the previous bullet point, but that is what the previous DoD includes:
>Logging/monitoring is configured, and it both records the metrics defined below and alerts the development team when the API is down per the [API Logging & Monitoring milestone](https://github.com/HHS/grants-equity/issues/370)

I'll change this to say:
>Logging/monitoring is configured, and it both records the metrics defined below and alerts the development team when the API is down or _**other key monitoring criteria are met**_ per the [API Logging & Monitoring milestone](https://github.com/HHS/grants-equity/issues/370)